### PR TITLE
Add bulk custom django command

### PR DESCRIPTION
**Pull request by Ali-D-Akbar**

### CODE REVIEW

File: books/management/commands/add_bulk_books.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+import json
```
Comment (danialmalik) [inline books/management/commands/add_bulk_books.py]: Please sort the imports.

File: books/models.py [hunk 2]
```diff
@@ -1,10 +1,49 @@
...
     description = models.CharField(max_length=1000)
     user = models.ForeignKey(get_user_model(), on_delete=models.CASCADE)
+
+
+@receiver(post_save, sender=Book)
+def send_email(sender, instance, **kwargs):
+    port = 465  # For SSL
+    sender_email = "AliAkbarEdly@gmail.com"
+    receiver_email = get_user_model().objects.get(pk=instance.user_id).email
+    sender_password = "454lotte"
```
Comment (danialmalik) [inline books/models.py]: Never use hard coded credentials. Use config or .env files or environment variables for that.

File: books/management/commands/add_bulk_books.py (hunk 1 above)
Comment (Ali-D-Akbar) [reply books/management/commands/add_bulk_books.py]: done

File: books/models.py (hunk 2 above)
Comment (Ali-D-Akbar) [reply books/models.py]: The issue has been resolved. I've used .env file for that

File: bookclub/settings.py [hunk 3]
```diff
@@ -120,3 +123,7 @@
 # https://docs.djangoproject.com/en/2.2/howto/static-files/
 
 STATIC_URL = '/static/'
+
+# Create .env file path.
+env_path = Path('.') / '.env'
+load_dotenv(dotenv_path=env_path)
```
Comment (danialmalik) [inline bookclub/settings.py]: Good.

**Pull request merged**